src/types/onyx/BillingReceiptDetails.ts

```diff
@@ -0,0 +1,61 @@
+import type * as OnyxCommon from './OnyxCommon';
+
+/**
+ * Billing item
+ */
+type BillingItem = {
+    /** The description of the item */
+    description: string;
+
+    /** The amount associated with the item */
+    amount: number;
+
+    /** The formatted string representation of the amount */
+    formattedAmount: string;
+
+    /** The type/category of the item */
+    type: string;
+
+    /** The percentage related to the item */
+    percentage: number;
+};
+
+/**
+ * Billing receipt details model
+ */
+type BillingReceiptDetails = {
+    /** The total amount before any discounts are applied */
+    subtotal: number;
+
+    /** A list of receipts that do not have discounts applied */
+    receiptsWithoutDiscount: BillingItem[];
+
+    /** A list of discounts applied to the billing */
+    discounts: BillingItem[];
+
+    /** A list of sales tax items */
+    salesTax: BillingItem[];
```

Comment:
The `salesTax` value is currently an object, but pressing 'Delete' from the overview causes the app to crash.
```
{
    "description": "Sales Tax ( - 0%)",
    "amount": 104,
    "formattedAmount": "$1.04",
    "type": "",
    "percentage": 0
}
```